toolbox

```diff
@@ -1033,29 +1042,25 @@ enter()
 
     set_environment=$(create_environment_options)
 
-    echo "$base_toolbox_command: looking for $SHELL in container $toolbox_container" >&3
-
-    if $prefix_sudo podman exec "$toolbox_container" test -f "$SHELL" 2>&3; then
-        shell_to_exec="$SHELL"
-    else
-        echo "$base_toolbox_command: $SHELL not found in $toolbox_container; using $shell_to_exec instead" >&3
+    if [ "$fallback_command" != "" ]; then
+	binary=$(echo "$command" | cut -d ' ' -f 1)
```

Comment:
It would be more robust to let the shell split the command string into an array of arguments, than doing it ourselves with `cut(1)`. Imagine cases where the command has spaces or backslash escapes.

We should also be more pedantic in `run` and disable the smarts introduced for #128.  It's expected that users might use `run` to create ad-hoc *.desktop files, and user interaction wouldn't make sense when used like that. It's also a more advanced command than `enter` and new users are less likely to use it. So it shouldn't affect usability.